src/test/groovy/io/seqera/wave/test/RedisTestContainer.groovy

```diff
@@ -52,6 +52,13 @@ trait RedisTestContainer {
         redisContainer.getMappedPort(6379)
     }
 
+    def setupSpec() {
+        if (!redisContainer.isRunning()) {
+            redisContainer.start()
+            log.debug "Started Redis test container"
```

Comment:
Wonder if not [all the snippet](https://github.com/seqeralabs/wave/pull/436/files#diff-6bcc5bde499446dc5a43bfc08d5bbd35aeda6055588d2dcb56fb52a417ce3c9bR38-R43) in the static block should be moved here

Reply:
We can do that, too, as the point of this change is to make sure that whenever a new test class initiates, it checks whether the Redis is running or not. If it doesn't, then start it. We can go with this; if there are still failures, we can try adding everything in setupspec.